install_config/configuring_azure.adoc

```diff
@@ -106,4 +106,37 @@ Therefore, *_node-config.yaml_* should be in *_/etc/origin/node_* instead of
 *_/etc/_*.
 ====
 
-. Start or restart the {product-title} services on the master and all nodes.
+[[azure-applying-configuration-changes]]
+== Applying Configuration Changes
+
+Start or restart {product-title} services on all master and node hosts to apply your
+configuration changes:
+
+ifdef::openshift-enterprise[]
+----
+# systemctl restart atomic-openshift-master
+# systemctl restart atomic-openshift-node
+----
+endif::[]
+ifdef::openshift-origin[]
+----
+# systemctl restart origin-master
+# systemctl restart origin-node
+----
+endif::[]
+
+Switching from not using a cloud provider to using a cloud provider produces an
+error message. Adding the cloud provider tries to delete the node because the
+node switches from using the *hostname* as the `*externalID*` (which would have
+been the case when no cloud provider was being used) to using the Azure
+`*instance-id*` (which is what the Azure cloud provider specifies). To resolve
+this issue:
+
+.  Log in to the CLI as a cluster administrator.
+.  Delete the nodes:
++
+----
+$ oc delete node <node_name>
+----
+.  On each node host, restart the *atomic-openshift-node* service.
```

Comment:
How about changing atomic-openshift-node service to {product-title} node service?

Reply:
`atomic-openshift-node` is the actual name of service therefore we cannot replace it with {product-title}.

Reply:
OK.
Just per the first paragraph of https://github.com/gaurav-nelson/openshift-docs/blob/e511b0f626c73c349f8ec85a66059e696baa6e70/install_config/configuring_azure.adoc#azure-applying-configuration-changes